Code/Framework/AzFramework/Platform/Windows/AzFramework/Input/Devices/Mouse/InputDeviceMouse_Windows.cpp

```diff
@@ -137,18 +138,25 @@ namespace AzFramework
     {
         memset(&m_lastClientRect, 0, sizeof(m_lastClientRect));
 
-        if (s_instanceCount++ == 0)
+        if (!s_instanceCount)
+        {
+            s_instanceCount = AZ::Environment::CreateVariable<int>("InputDeviceMouseInstanceCount", 0);
+        }
+
+        int instanceCount = s_instanceCount.Get();
+        if (instanceCount++ == 0)
         {
             // Register for raw mouse input
             RAWINPUTDEVICE rawInputDevice;
             rawInputDevice.usUsagePage = RAW_INPUT_MOUSE_USAGE_PAGE;
-            rawInputDevice.usUsage     = RAW_INPUT_MOUSE_USAGE;
-            rawInputDevice.dwFlags     = 0;
-            rawInputDevice.hwndTarget  = 0;
+            rawInputDevice.usUsage = RAW_INPUT_MOUSE_USAGE;
+            rawInputDevice.dwFlags = 0;
+            rawInputDevice.hwndTarget = 0;
             const BOOL result = RegisterRawInputDevices(&rawInputDevice, 1, sizeof(rawInputDevice));
             AZ_Assert(result, "Failed to register raw input device: mouse");
             AZ_UNUSED(result);
         }
+        s_instanceCount.Set(instanceCount);
```

Comment:
As above, if we do need to reset the environment variable, we should probably do that at the end of the 'if' block above, then wrap this in an 'else' statement.